database/models/User.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  DataType,
+  Column,
+  Model,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+  BelongsToMany,
+} from "sequelize-typescript"
+
+import { Site } from "./Site"
+import { SiteMember } from "./SiteMember"
+
+@Table({ tableName: "users" })
+export class User extends Model {
+  @Column({
+    autoIncrement: true,
+    primaryKey: true,
+    allowNull: false,
+    type: DataType.BIGINT,
+  })
+  id!: number
+
+  @Column({
+    allowNull: true,
+    unique: true,
+    type: DataType.TEXT,
+    validate: {
+      isEmail: true,
+    },
+  })
+  email!: string
+
+  @Column({
+    allowNull: false,
+    unique: true,
+    type: DataType.TEXT,
+    validate: {
+      notEmpty: true,
+    },
+  })
+  githubId!: string
+
+  @Column({
+    allowNull: true,
+    type: DataType.STRING(255),
+  })
```

Comment:
we should either 1. have the same validator as [here](https://github.com/isomerpages/isomercms-backend/pull/364/files#diff-f5a20552f742c7a8229e2a99ba078eaee60527809c656c1678f0cbba5458bfebR49-R50) or 2. remove validation entirely (we allow everything) 

i think we should just use the validator as per the first point and actually stricten the check in here/the routes to sg only by specifying the [options](https://github.com/validatorjs/validator.js).

Reply:
Will fix in a separate issue/PR.